nil/services/rpc/jsonrpc/txpool_api.go

```diff
@@ -0,0 +1,69 @@
+package jsonrpc
+
+import (
+	"context"
+
+	"github.com/NilFoundation/nil/nil/common/logging"
+	"github.com/NilFoundation/nil/nil/internal/types"
+	"github.com/NilFoundation/nil/nil/services/rpc/rawapi"
+)
+
+type TxPoolStatus struct {
+	Pending uint64 `json:"pending"`
+	Queued  uint64 `json:"queued"`
+}
+
+type TxPoolContent struct {
+	Pending map[string]map[string]*types.Transaction `json:"pending"`
```

Comment:
Let's use jsonrpc.RPCTransaction type instead of types.Transaction. It will be consistent with other functions.

Erigon also uses RPCtransaction type in their implementation https://github.com/erigontech/erigon/blob/df85a9a43f899a69a3c3ce7b894364c0891dd662/turbo/jsonrpc/eth_api.go#L418